modules/nf-core/taxpasta/merge/meta.yml

```diff
@@ -0,0 +1,56 @@
+name: "taxpasta_merge"
+description: Standardise and merge two or more taxonomic profiles into a single table
+keywords:
+  - taxonomic profile
+  - standardise
+  - standardisation
+  - metagenomics
+  - taxonomic profiling
+  - otu tables
+  - taxon tables
+tools:
+  - "taxpasta":
+      description: "TAXonomic Profile Aggregation and STAndardisation"
+      homepage: "https://taxpasta.readthedocs.io/"
+      documentation: "https://taxpasta.readthedocs.io/"
+      tool_dev_url: "https://github.com/taxprofiler/taxpasta"
+      doi: ""
+      licence: "['Apache-2.0']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - profile:
+      type: file
+      description: A list of taxonomic profiler output files (typically in text format, mandatory)
+      pattern: "*.{tsv,csv,arrow,parquet,biom}"
+  - samplesheet:
+      type: file
+      description: Directory containing at a minimum nodes.dmp and names.dmp files (optional)
```

Comment:
Copy paste error. But when updateing it, make sure to mention that it needs paths to be locally from the work directory, and that you still need to provide the `profiles` even if you give a samplesheet

Reply:
Oops 😁

Reply:
Do we need to mention that the paths need to be locally from the work directory? I thought this was just the case to run the tests.

Reply:
Yes it does need to be mentioned! It is'nt just the tests that apply, any use of the module.